src/ORM/Locator/TableLocator.php

```diff
@@ -78,6 +94,40 @@ public function __construct(?array $locations = null)
         }
     }
 
+    /**
+     * Set if fallback class should be used.
+     *
+     * Controls whether a fallback class should be used to create a table
+     * instance if a concrete class for alias used in `get()` could not be found.
+     *
+     * @param bool $allow Flag to enable or disable fallback
+     * @return $this
+     */
+    public function allowFallbackClass(bool $allow)
```

Comment:
We already have `I18n::useFallback()`, that's why I had chose "use" prefix for consistency.

Reply:
allowFallbackClass feels correct, but if useFallback is strongly preferred then I won't object.